x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/parser/ExpressionTests.java

```diff
@@ -373,32 +373,47 @@ public void testUnquotedIdentifiers() {
     public void testDurationLiterals() {
         int value = randomInt(Integer.MAX_VALUE);
```

Comment:
Super nit: Maybe we should use a random long instead to better cover the relevant range (esp. for nanos).

Reply:
IIRC some of the static Duration/Period methods fail with too large values and this was the reason to use an int here.